### PR TITLE
[12.0][MIG] analytic wip report extend

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
pleasereview

File: analytic_wip_report_extend/tests/test_analytic_wip_report_extend.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Copyright 2015 Eficent Business and IT Consulting Services S.L.
```
Comment (MiquelRForgeFlow) [inline analytic_wip_report_extend/tests/test_analytic_wip_report_extend.py]:
```suggestion
# Copyright 2019 Eficent Business and IT Consulting Services S.L.
```

File: analytic_wip_report_extend/tests/__init__.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Copyright 2015 Eficent Business and IT Consulting Services S.L.
```
Comment (MiquelRForgeFlow) [inline analytic_wip_report_extend/tests/__init__.py]: delete line

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

File: analytic_wip_report_extend/tests/test_analytic_wip_report_extend.py (hunk 1 above)
Comment (AaronHForgeFlow) [reply analytic_wip_report_extend/tests/test_analytic_wip_report_extend.py]: why? the code is that old

Comment (AaronHForgeFlow) [reply analytic_wip_report_extend/tests/test_analytic_wip_report_extend.py]: ok. It does not matter, i just did

Comment (MiquelRForgeFlow) [reply analytic_wip_report_extend/tests/test_analytic_wip_report_extend.py]: this test is new!

**Comment by AaronHForgeFlow** [comment]
skeleton2

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**